File: NovaCore-Spigot_v1_18_R2/src/main/java/net/zeeraa/novacore/spigot/version/v1_18_R2/VersionIndependantItemsImplementation.java
package net.zeeraa.novacore.spigot.version.v1_18_R2;

import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;

public class VersionIndependantItemsImplementation extends net.zeeraa.novacore.spigot.abstraction.VersionIndependentItems {
	@Override
	public ItemStack getPlayerSkull() {
		return new ItemStack(Material.PLAYER_HEAD, 1);
	}

	@Override
	public boolean isPlayerSkull(ItemStack item) {
		return item.getType() == Material.PLAYER_HEAD || item.getType() == Material.PLAYER_WALL_HEAD;
	}
}

File: NovaCore-Spigot_v1_19_R3/src/main/kotlin/net/zeeraa/novacore/spigot/version/v1_19_R3/VersionIndependantItemsImplementation.kt
package net.zeeraa.novacore.spigot.version.v1_19_R3

import net.zeeraa.novacore.spigot.abstraction.VersionIndependentItems
import org.bukkit.Material
import org.bukkit.inventory.ItemStack

class VersionIndependantItemsImplementation : VersionIndependentItems() {
	override fun getPlayerSkull(): ItemStack {
		return ItemStack(Material.PLAYER_HEAD, 1)
	}

	override fun isPlayerSkull(item: ItemStack): Boolean {
		return item.type == Material.PLAYER_HEAD || item.type == Material.PLAYER_WALL_HEAD
	}
}